### DISCUSSION TITLE
Please add Oppo to the Buggy Samsung Workaround!

### DISCUSSION THREAD

**Discussion by Miquest** [NightscoutFoundation/xDrip#2697] [Q&A]
@jamorham ,

please add also Oppo to the Buggy Samsung Workaround!
I have the same problems like the users who are on Oneplus phones, I think it could also work for me!
I'm able to test it and would give you feedback for the feature.

**Comment by Miquest** [comment]
@jamorham, @Navid200 ?

**Reply by Navid200** [reply]
@Miquest Do you see slowest wake note in red on the system status page on that phone?

I prefer to add this for all phones.  But, it needs some work.

**Reply by Miquest** [reply]
I'm using the BYODA, which is broadcasting the data to xDrip Plus, as workaround. But before, I had a slowest wake of about 6 minutes. I thought we could add it to the Buggy Samsung Workaround such as Oneplus a few weeks before. I could give a report if it works or not

**Reply by Miquest** [reply]
Isn't just one line of code needed to add a vendor to the workaround?
